build/template_kql/database/database_sqldb_resources_details_summary.kql

```diff
@@ -0,0 +1,4 @@
+${extend_resource}
+| project Type, SkuName, Kind, SubscriptionId, Name, State, Location, ResourceGroup, Zones, AvailabilityZone, AvailabilitySet, Capacity, FaultDomain, OSDisk, ReadReplica, AutomaticFailover, MultipleWriteLocations, StorageAutogrow, ReplicationRole, Backup = tolower(Backup), Environment
+| where Type == 'microsoft.sql/servers/databases' and SkuName !contains "datawarehouse" and Kind !contains "system"
+| summarize ResourceTotal=count(Name), NotRunning=dcountif(Name, State !in~ ("online", "paused")), NotRecommendedSKU=dcountif(Name, SkuName in~ ("standard", "basic")), NoAVZone=dcountif(Name, AvailabilityZone == "not configured"), NoGRSBackup=dcountif(Name, OSDisk !~ "grs")
```

Comment:
Is there a way to filter further on the NotRecommendedSKU condition
- Basic to be there.
- Standard S0,S1,S2 to be there. We do allow S3 and above for prod: https://learn.microsoft.com/en-us/azure/azure-sql/database/service-tiers-dtu?view=azuresql#compare-service-tiers

Check the important section in this document. 

This is not just for summary, maybe we need to update for details section as well.

Also UI label for this field can be changed from "Not Production SKU" to "Non-production SKU"

Reply:
I've tried that.
![image](https://github.com/Azure/reliability-workbook/assets/3822284/a33bf5c9-fdeb-4f57-9f36-b7d3bbb20c82)